ajax/libs/simpleCookie/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "simplecookie",
+  "version": "1.0.1",
```

Comment:
Please don't include this attr, the bot will populate this itself when it pulls in the version files.